lua/starfall/libs_sh/holograms.lua

```diff
@@ -189,6 +198,17 @@ function holograms_library.create(pos, ang, model, scale)
 	end
 end
 
+--- Removes all holograms created by the calling chip
```

Comment:
This one still need to be removed

Reply:
Oh slipped past me. I'll remove and test it again